jquery.signaturepad.js

```diff
@@ -339,11 +351,18 @@ function SignaturePad (selector, options) {
    * @param {Object} touchObject The object context registered to the event; canvas
    */
   function startDrawing (e, touchObject) {
+	if($(document).height() > $(window).height())
+		document.body.style.overflowY = "scroll";
+	
+	  lastScrollY = window.scrollY;
+	  document.body.style.position = 'fixed';
```

Comment:
I had to fix the body or the stylus would scroll the page up and down :disappointed: